test/unit_test.c

```diff
@@ -2543,16 +2543,21 @@ static void test_udp(void) {
 }
 
 static void test_check_ip_acl(void) {
-  uint32_t ip = mg_htonl(0x01020304);
-  ASSERT(mg_check_ip_acl(mg_str(NULL), ip) == 1);
-  ASSERT(mg_check_ip_acl(mg_str(""), ip) == 1);
-  ASSERT(mg_check_ip_acl(mg_str("invalid"), ip) == -1);
-  ASSERT(mg_check_ip_acl(mg_str("+hi"), ip) == -2);
-  ASSERT(mg_check_ip_acl(mg_str("+//"), ip) == -2);
-  ASSERT(mg_check_ip_acl(mg_str("-0.0.0.0/0"), ip) == 0);
-  ASSERT(mg_check_ip_acl(mg_str("-0.0.0.0/0,+1.0.0.0/8"), ip) == 1);
-  ASSERT(mg_check_ip_acl(mg_str("-0.0.0.0/0,+1.2.3.4"), ip) == 1);
-  ASSERT(mg_check_ip_acl(mg_str("-0.0.0.0/0,+1.0.0.0/16"), ip) == 0);
+  const uint32_t ip4 = mg_htonl(0x01020304);
+  struct mg_addr ip;
+  memset(&ip, 0, sizeof(ip));
+  memcpy(ip.ip, &ip4, sizeof(ip4));
```

Comment:
Change these 4 lines to:

```c
struct mg_addr ip = {{1,2,3,4}, 0, false};  // 1.2.3.4
```

Reply:
I was sure the compiler would complain about a smaller char array and didn't even try...